packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -76,20 +77,16 @@ export default class PrayerRequest extends RockApolloDataSource {
 
   // QUERY ALL PrayerRequests
   getAll = async () => {
-    try {
```

Comment:
Not sure how removing the `try ... catch` block makes this better. Maybe we should just make it a better error message like we have in other places?

Reply:
we need to use bugsnag for error message, not console.log, but we can't do that until it's in the API repo